content/40.inequality.md

```diff
@@ -144,13 +212,16 @@ Furthermore, cell phone GPS data suggests that lower socioeconomic status may al
 
-A preprint [@doi:10.1101/2020.04.05.20054502] provided observational evidence that geographical areas in the United States that suffer from worse air pollution by fine particulate matter have also suffered more COVID-19 deaths per capita, after adjusting for demographic covariates.
-Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., [@doi:10.1073/pnas.1803222115]).
+Environmental disparities may also influence differences in COVID-19 outcomes.
+A preprint [@doi:10.1101/2020.04.05.20054502] provided observational evidence that geographical areas in the United States that suffer from worse air pollution by fine particulate matter have also suffered more COVID-19 deaths per capita, after adjusting for demographic covariates.<!--To Do: update reference: 10.1126/sciadv.abd4049 -->
+Although a lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., [@doi:10.1073/pnas.1803222115]).
 Exposure to air pollution is associated with both poverty (e.g., [@doi:10.3390/ijerph15061114]) and chronic inflammation [@doi:10.1016/j.neubiorev.2018.06.002].
 Other outcomes of environmental racism, such as the proximity of abandoned uranium mines to Navajo land, can also cause respiratory illnesses and other health issues [@url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416].
-Similarly, preliminary findings indicate that nutritional status (e.g., vitamin D deficiency [@doi:10.1210/clinem/dgaa733]) may be associated with COVID-19 outcomes, and reduced access to grocery stores and fresh food often co-occurs with environmental racism [@url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416; @doi:10.1056/NEJMp2021264].
-Taken together, the evidence suggests that low-income workers who face greater exposure to SARS-CoV-2 due to their home or work conditions are also more likely to face environmental and social stressors associated with increased inflammation, and therefore with increased risk from COVID-19.
-In particular, structural racism can play an important role on disease severity after SARS-CoV-2 exposure, due to consequences of racism which include an increased likelihood of poverty and its associated food and housing instability.
+Similarly, preliminary findings indicate that nutritional status (e.g., vitamin D deficiency and other nutritional deficiencies [@individual-nutraceuticals]) may be associated with COVID-19 outcomes, and reduced access to grocery stores and fresh food often co-occurs with environmental racism [@url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416; @doi:10.1056/NEJMp2021264].
```

Comment:
perhaps this sentence would fit better in the section above that discusses the link between poverty and inflammation (so right before line 211, before the discussion of environmental racism)